src/components/Card.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+
+function Card(props) {
+    const data = props.data;
```

Comment:
This could be a great use case for destructuring `const {title, date, etc} = props.data`